nodes/mtnode.py

```diff
@@ -71,17 +74,41 @@ def __init__(self):
 
         self.frame_local = get_param('~frame_local', 'ENU')
 
-        self.diag_msg = DiagnosticArray()
+
         self.stest_stat = DiagnosticStatus(name='mtnode: Self Test', level=1,
                                            message='No status information')
         self.xkf_stat = DiagnosticStatus(name='mtnode: XKF Valid', level=1,
                                          message='No status information')
         self.gps_stat = DiagnosticStatus(name='mtnode: GPS Fix', level=1,
                                          message='No status information')
-        self.diag_msg.status = [self.stest_stat, self.xkf_stat, self.gps_stat]
+
+	self.device_info = {
```

Comment:
Never mix tab and space in Python code!
This code uses four spaces for indentation (as advised in [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation) please comply to this (you have several lines with tabulations).